esports_data_from_rimble/Makefile

```diff
@@ -57,12 +57,6 @@ team-kill-diff: check build
 clean:
 	@rm -rf tmp
 
-.PHONY: update-rimble-test-data
```

Comment:
Since we don't pin our examples to a specific match id anymore I got rid of this convenience function. This really only needs to be updated when the match status response format changes anyway, which should be very infrequent